src/flows/context/flow.query.tsx

```diff
@@ -144,62 +144,67 @@ export const FlowQueryProvider: FC = ({children}) => {
   }, [variables, flow?.range])
 
   const generateMap = (withSideEffects?: boolean): Stage[] => {
-    const stages = flow.data.allIDs.reduce((acc, panelID) => {
-      const panel = flow.data.byID[panelID]
+    try {
```

Comment:
wrapped this in a try/catch since I've come across situations where this'll crash the app, which is bad. I don't know if the way we're handling the error is good since it's symptomatic that something with the query generation is wrong, but it'll do for now

Reply:
rats that's not good. Do you mean you've seen this happen with the yjs integration specifically? Or like in general? Could we send these errors to honeybadger so we can see how often it happens and try to look into it down the road?